app/containers/uri/URILandingPage.js

```diff
@@ -23,10 +24,17 @@ export default class URILandingPage extends Component<InjectedOrGenerated<Genera
 
   onConfirm: void => void = () => {
     const { wallets } = this.generated.stores;
+
+    const firstShelley = wallets.allWallets.find(publicDeriver => {
```

Comment:
@ashisherc I would update this to use a more descriptive name `firstShelley` is strange

Reply:
done